### ISSUE TITLE
Czy to jakaś przerwa w usłudze?

### ISSUE THREAD

**Issue by SobiszeK** [epforgpl/_mojePanstwo-Portal#758]
Witam,
wyszukiwarka działa poprawnie, jendakże wizualizacja nie chcę się pokazać zawsze wyskakuje błąd:
"Żądana strona nie jest obecnie dostępna"

**Comment by danielmacyszyn** [comment]
Tak. Mamy w tej chwili problem z jednym z serwerów baz danych. Pracujemy
nad naprawą...

pt., 30 cze 2017 o 19:32 użytkownik Kamil Sobiszewski <
notifications@github.com> napisał:
